helpers/clientopts.go

```diff
@@ -1,10 +1,10 @@
 package helpers
 
 type RegistrationOpts struct {
-	Localpart string // default '' (don't care)
-	DeviceID  string // default '' (generate new)
-	Password  string // default 'complement_meets_min_password_requirement'
-	IsAdmin   bool   // default false
+	LocalpartSuffix string // default '' (don't care)
```

Comment:
Can we make this a prefix so that I can grep for `@alice` instead of `@[^:]+alice`?

Reply:
c.f. https://github.com/matrix-org/sliding-sync/blob/57d8c1deba918597f558b013cbc94432ed14d649/tests-e2e/main_test.go#L214

Reply:
I guess. It's just that there are restrictions on valid characters for `localpart[0]` so it adds arbitrary restrictions on test writers.